app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -272,34 +272,125 @@ function handleKeyUp(event) {
-        // Get user details and extract code of conduct & comment count
-        var user = userData();
-        if (!user) {
-            return;
-        }
+// Handler for when Ctrl+Enter OR Command+Enter is pressed
+function handleSubmit(event) {
+  // Get user details and extract code of conduct & comment count
+  var user = userData();
+  if (!user) {
+      return;
+  }
 
-        var commentCount = user.number_of_comments;
-
-        // var codeOfConduct = user.checked_code_of_conduct;
-        // Code of conduct is false the first time even if the user has accepted it
-        // so we are not using it in the check below
-        // It also seems unnecessary if comment count >= 1 indicates that
-        // it was previously accepted
-
-        // Ensure that user has commented at least once
-        // and we have a non-empty comment to post
-        if (commentCount >= 1 &&
-            event.target.value.trim() !== '') {
-            // get a reference to the submit button for the text area (first level comment vs. reply comment)
-            var parentFormId = event.target.parentElement.parentElement.id || event.target.parentElement.id;
-            var submitButton = document.querySelector('#' + parentFormId + ' input[type="submit"].comment-action-button');
-            submitButton.click();
-        }
+  var commentCount = user.number_of_comments;
+
+  // var codeOfConduct = user.checked_code_of_conduct;
+  // Code of conduct is false the first time even if the user has accepted it
+  // so we are not using it in the check below
+  // It also seems unnecessary if comment count >= 1 indicates that
+  // it was previously accepted
+
+  // Ensure that user has commented at least once
+  // and we have a non-empty comment to post
+  if (commentCount >= 1 &&
+    event.target.value.trim() !== '') {
+    // get a reference to the submit button for the text area (first level comment vs. reply comment)
+    var parentFormId = event.target.parentElement.parentElement.id || event.target.parentElement.id;
+    var submitButton = document.querySelector('#' + parentFormId + ' input[type="submit"].comment-action-button');
+    submitButton.click();
+  }
+}
+
+// Handler for when Ctrl+B/I OR Command+B/I is pressed
+function handleBoldAndItalic(event) {
+  var textArea = event.target;
+
+  // Check if this function is supported before using it
+  // All new browsers support it
+  if(textArea.setRangeText) {
+    var selection = textArea.value.substring(textArea.selectionStart, textArea.selectionEnd);
+    var selectionStart = textArea.selectionStart;
+    var surroundingStr =  event.keyCode === KEY_CODE_B ? "**" : "_";
+
+    replaceSelectedText(textArea, `${surroundingStr}${selection}${surroundingStr}`);
+
+    var selectionStartWithOffset = selectionStart + surroundingStr.length;
+    textArea.setSelectionRange(selectionStartWithOffset, selectionStartWithOffset + selection.length);
+  }
+}
+
+// Handler for when Ctrl+B/I OR Command+B/I is pressed
```

Comment:
I think you meant to say Ctrl+K or Command+I here.

Reply:
Yes, thanks. Got to stop with the 4 AM checkins, LOL.